### PR TITLE
chore: working on workflows to mitigate the issues

### PR DESCRIPTION
**Pull request by sandeep-deriv**
## Changes:
Updated following GitHub workflows

- `build_and_push_docker_image/action.yml`
- `send_slack_notifications/action.yml`
- `versioning/action.yml`

### Screenshots:

NA

### CODE REVIEW

**Review by niloofar-deriv — APPROVED** [review]

File: .github/actions/build_and_push_docker_image/action.yml [hunk 1]
```diff
@@ -36,31 +36,51 @@ runs:
         echo "NAMESPACE=${{ inputs.K8S_NAMESPACE }}" >> "$GITHUB_ENV"
         echo "KUBE_SERVER=${{ inputs.KUBE_SERVER }}" >> "$GITHUB_ENV"
         echo "SERVICEACCOUNT_TOKEN=${{ inputs.SERVICEACCOUNT_TOKEN }}" >> "$GITHUB_ENV"
```
Comment (ritik-deriv) [inline .github/actions/build_and_push_docker_image/action.yml:38]: We need to remove all of these as well. @sandeep-deriv As discussed, we don't need this step at all.

File: .github/actions/build_and_push_docker_image/action.yml [hunk 2]
```diff
@@ -36,31 +36,51 @@ runs:
...
+
     - name: Verify nginx image
+      env:
+        GITHUB_REF_NAME: ${{ github.ref_name }}
+        DOCKERHUB_ORGANISATION: ${{ inputs.DOCKERHUB_ORGANISATION }}
+        APP_NAME: deriv-static-bot
       run: |
         set -e
-        docker run --rm ${DOCKERHUB_ORGANISATION}/${APP_NAME}:${{ github.ref_name }} nginx -t
+        docker run --rm $DOCKERHUB_ORGANISATION/$APP_NAME:GITHUB_REF_NAME nginx -t
```
Comment (ritik-deriv) [inline .github/actions/build_and_push_docker_image/action.yml:57]: Missing `$` here in GITHUB_REF_NAME

File: .github/actions/build_and_push_docker_image/action.yml [hunk 3]
```diff
@@ -36,31 +36,51 @@ runs:
...
+        GITHUB_REF_NAME: ${{ github.ref_name }}
+        CA_CRT: ${{ inputs.CA_CRT }}
+        APP_NAME: deriv-static-bot
       run: |
         git clone https://github.com/binary-com/devops-ci-scripts
         cd devops-ci-scripts/k8s-build_tools
         echo $CA_CRT | base64 --decode > ca.crt
         export CA="ca.crt"
-        ./release.sh ${APP_NAME} ${{ github.ref_name }}
+        ./release.sh $APP_NAME $GITHUB_REF_NAME
```
Comment (ritik-deriv) [inline .github/actions/build_and_push_docker_image/action.yml:85]: This script might require other env variables like Namespace and all. You can add them to this step if needed.

[Ref](https://github.com/binary-com/devops-ci-scripts/blob/1a9cb4bdac78a9dc722a713df10cf224808a3899/k8s-build_tools/release.sh#L40)

File: .github/actions/build_and_push_docker_image/action.yml (hunk 2 above)
Comment (sandeep-deriv) [reply .github/actions/build_and_push_docker_image/action.yml:57]: this is done already 👍

**Pull request merged**